packages/slice-machine/lib/builders/SliceBuilder/Header/VariationsPopover/index.tsx

```diff
@@ -6,7 +6,7 @@ import MenuList from "./MenuList";
 
 import { Button, Box, ThemeUICSSObject, ThemeUIStyleObject } from "theme-ui";
 
-import { RiArrowDropDownLine } from "react-icons/ri";
+import { RiArrowDropDownFill, RiArrowDropUpFill } from "react-icons/ri";
 
 const VarationsPopover: React.FunctionComponent<{
```

Comment:
Seems the same as the dropdownMenu you made.

Reply:
Yeah but this one has some more features specific to the variations changer like creating more variations in the slice builder. It's a bit of a shame :/